go.mod

```diff
@@ -17,4 +17,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 	gotest.tools v2.2.0+incompatible
+	honnef.co/go/tools v0.0.1-2019.2.3
```

Comment:
Is this used? and when I run `go mod tidy`,  `go.mod` is shrunk.
Do we need all of these dependencies?

```
--- a/go.mod
+++ b/go.mod
@@ -4,18 +4,11 @@ go 1.13

 require (
        github.com/go-yaml/yaml v2.1.0+incompatible
-       github.com/google/go-cmp v0.4.0 // indirect
        github.com/kr/pretty v0.2.0 // indirect
        github.com/mitchellh/go-homedir v1.1.0
-       github.com/pkg/errors v0.9.1 // indirect
-       github.com/screwdriver-cd/launcher v6.0.17+incompatible
        github.com/sirupsen/logrus v1.4.2
        github.com/spf13/cobra v0.0.5
        github.com/stretchr/testify v1.4.0
-       golang.org/x/lint v0.0.0-20200130185559-910be7a94367 // indirect
-       golang.org/x/tools v0.0.0-20200206050830-dd0d5d485177 // indirect
        gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
        gopkg.in/yaml.v2 v2.2.8 // indirect
-       gotest.tools v2.2.0+incompatible
-       honnef.co/go/tools v0.0.1-2019.2.3
 )
```

Reply:
I see.
I fixed it